mlcomp/parallelm/components/restful/uwsgi_statistics.py

```diff
@@ -82,7 +82,13 @@ def _read_raw_statistics(self):
         sock = self._setup_stats_connection()
         if sock:
             try:
-                data = sock.recv(UwsgiStatistics.STATS_JSON_MAX_SIZE_BYTES)
+                data = b""
```

Comment:
Instead, for much better performance, it's better to use list & join (in python strings are immutable and therefore each '+' is involved in delete and creation of new string:

```suggestion
                parts = []
                while True:
                    buff = sock.recv(UwsgiStatistics.STATS_JSON_MAX_SIZE_BYTES)
                    parts.append(buff)
                    if len(buff) < UwsgiStatistics.STATS_JSON_MAX_SIZE_BYTES:
                        break
                data = b''.join(parts)
```